doc/version_history.rst

```diff
@@ -6,6 +6,20 @@
 Version History
 ###############
 
+v0.3.0
```

Comment:
Does this really require a minor release? wouldn't a v0.2.1 do?

Reply:
I think so because it changes the intended behavior. It is not a bug fix.